node/Cargo.toml

```diff
@@ -30,17 +30,20 @@ tokio = { workspace = true }
 
 # Local
 pallet-file-system = { workspace = true }
+pallet-proofs-dealer = { workspace = true }
 storage-hub-runtime = { workspace = true }
 storage-hub-infra = { workspace = true }
 file-manager = { workspace = true }
 forest-manager = { workspace = true }
 shc-common = { workspace = true }
 shc-rpc = { workspace = true }
+shp-file-key-verifier = { workspace = true }
+storage-hub-traits = { workspace = true }
 
 clap = { workspace = true }
 log = { workspace = true }
 codec = { workspace = true }
-serde = { workspace = true }
+serde = { workspace = true, default-features = true }
```

Comment:
You removed the `default-features = false` in the root `Cargo.toml`. If you do that, wouldn't this be redundant?

Reply:
I removed it in root so that we can import it in runtime with `no-std`.